api-reference/integrations/update-integration.mdx

```diff
@@ -1,3 +1,75 @@
 ---
 openapi: put /v1/integrations/{integrationId}
----
\ No newline at end of file
+---
+
+<RequestExample>
+
+```javascript Node.js
+import { Novu } from '@novu/node';  
+export const novu = new Novu('<REPLACE_WITH_API_KEY_FROM_ADMIN_PANEL>');
+await novu.integrations.getWebhookProviderStatus(ProvidersIdEnum.SendGrid);
```

Comment:
I think the ProvidersIdEnum needs to be imported too else this code will throw an error